### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by subhashiniy** [sebastian-hofstaetter/transformer-kernel-ranking#4]
Hi, 
  Are there pre-trained models available for download?

Thanks in advance.

**Comment by sebastian-hofstaetter** [comment]
Hi,
I am sorry at the moment we don't have pre-trained models to download. But TK is quite efficient to train from scratch - if you have a single newer GPU it will only take a couple of hours to get very good results.
Best,
Sebastian

**Comment by seanmacavaney** [comment]
Any chance you could reconsider and release the pre-trained models?

**Comment by sebastian-hofstaetter** [comment]
Yeah I guess so :) which models would you be interested in? The original TK/TKL or the knowledge distilled ones?

**Comment by seanmacavaney** [comment]
Awesome- thanks! The original TK and TKL would be great.

**Comment by abheesht17** [comment]
Hello! Have the pretrained models for TK been released? If not, could you please share them? Would be of great help! :)
